### DISCUSSION TITLE
Office_S_Tips/pki/2019/09/19/Weryfikacja_podpisanego_PDF

### DISCUSSION THREAD

**Discussion by giscus** [andrzejQ/Office_S_Tips#17] [Komentarze]
# Office_S_Tips/pki/2019/09/19/Weryfikacja_podpisanego_PDF

Poprawność podpisu można zweryfikować w aplikacji do składania podpisu. W przypadku PDF można to też sprawdzić w darmowej aplikacji Adobe Reader DC.

https://andrzejq.github.io/Office_S_Tips/pki/2019/09/19/Weryfikacja_podpisanego_PDF.html

**Comment by andrzejQ** [comment]
Nowa wersja komentarzy z użyciem `giscus -> GitHub Discussions`.

**Comment by piotrber** [comment]
Powoli kończy się rok 2024 w kwestii weryfikacji dokumentu PDF podpisanego przy pomocy e-dowodu w Acrobat Readerze chyba się nic nie zmieniło...

**Reply by andrzejQ** [reply]
Podobno to jednak sprawa MSWiA - zob. tekst po rozwinięciu ">_w sprawie problemów z listą zaufania_". Napisałem zapytanie do **CPD MSWiA** ([Centrum Personalizacji Dokumentów MSWiA](https://www.gov.pl/web/cpd-mswia/)) i może dostanę jakąś odpowiedź dotyczącą terminu wpisania pl.ID na europejską listę TSL. Na razie pozostaje ręczne dodanie certyfikatu pl.ID jako zaufanego w Adobe.
